.github/workflows/all.yaml

```diff
@@ -44,7 +44,8 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17.x
+          # up this to 1.19.x when we update the version of golangci-lint to v1.48+
```

Comment:
Hm, I thought you do it the same way the k6 project does it ### where we *disable* plugins out of all plugins. In that case it just means that when I update golangci-lint it will add a bunch of new linters, likely a lot of them once we don't care about. 

But you seem to be doing in reverse - enabling only the ones you want.

If you all want I can update it in this PR :shrug:

Reply:
Yeah, we felt that the reverse strategy was more reliable for updates, since we have more control over which linters are enabled, and shouldn't run into any surprises. The drawback is that we likely miss some useful ones, but we can always ocasionally check if there are any new ones we'd like to enable.

Thanks for updating.

Reply:
Oh I do prefer your strategy and have an open (previous to this discussion) tab with a github issue on us moving to this strategy ;)